internal/provider/resource_cloudflare_access_application_test.go

```diff
@@ -103,6 +105,34 @@ func TestAccCloudflareAccessApplication_WithCORS(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareAccessApplication_WithSass(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_application.%s", rnd)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccessAccPreCheck(t)
+		},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckCloudflareAccessApplicationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudflareAccessApplicationConfigWithSaas(rnd, accountID),
```

Comment:
accountID is undefined here. you'll need `accountID := os.Getenv("CLOUDFLARE_ACCOUNT_ID")` to use it.

Reply:
I copied the syntax from other tests in this file (eg: `TestAccCloudflareAccessApplication_BasicAccount` & `TestAccCloudflareAccessApplication_WithADefinedIdps`) and they don't seem to define it either. Is that an issue too?

Reply:
looks like `resource_cloudflare_access_group_test.go` is defining `accountID` globally and the others are relying on it. definitely don't rely on this and instead define it at the test level.

Reply:
Understood, done.